src/components/AppBar.js

```diff
@@ -125,28 +129,22 @@ export default function SearchAppBar() {
               inputProps={{ 'aria-label': 'search' }}
             />
           </div>
-          {isLoggedIn ? (
+          {user && isLoaded ? (
             <div>
-              <IconButton
-                aria-label="account of current user"
-                aria-controls="menu-appbar"
-                aria-haspopup="true"
-                onClick={handleMenu}
-                color="inherit"
-              >
-                <AccountCircle />
-              </IconButton>
+              <ProfilePic handleMenu={handleMenu} />
```

Comment:
The icon button was the avatar icon (I think) which has been replaced by the profile picture to the handleMenu function has been moved into the replacement element/component.

Reply:
Nah so before the iconButton was a container for the icon itself, all it did was apply accessibility props to it essentially. Not a major diff really just means it will be invisible to screen readers

Reply:
Added a suggestion within profile pic that resolves this simply